src/Features/promo/components/Cosmetics/cosmetics.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Cosmetics } from './Cosmetics';
+
+jest.mock('next/link', () => {
+  return ({ children }: { children: React.ReactNode }) => children;
+});
+
+describe('<Cosmetics />', () => {
+  it('should render without crashing', () => {
```

Comment:
no-no: each test should have not only `render` function invocation but at least 1 `expect` function invocation. Otherwise, a test becomes useless. So, you can remove this one and leave the second one